languages/ruby/exercises/concept/ostruct/.docs/introduction.md

```diff
@@ -0,0 +1,33 @@
+Ruby comes with a Standard Library - a collection of classes for working with things such as dates, json, and networking.
+The also provide some useful functionality for making your code easier to work with.
+
+`OpenStruct` is part of the standard library and allows you to easily create an object from a hash. Rather than having to access using hash keys, `OpenStruct` instead allows us to use methods to access and set values.
```

Comment:
```suggestion
`OpenStruct` is part of the Standard Library and allows you to easily create an object from a hash. Rather than accessing `Hash` keys, `OpenStruct` instead allows us to use methods to access and set values.
```

Reply:
The [Ruby docs](https://ruby-doc.org/core-2.5.1/Hash.html) refer to "hash keys" as normal test, so I don't think we need to captialise/code-brace it.

```suggestion
`OpenStruct` is part of the Standard Library and allows you to easily create an object from a hash. Rather than having to access using hash keys, `OpenStruct` instead allows us to use methods to access and set values.
```